dev/CommonStyles/Slider_themeresources.xaml

```diff
@@ -206,7 +207,7 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="Slider">
-                    <Grid Margin="{TemplateBinding Padding}">
+                    <Grid Margin="{TemplateBinding Padding}" CornerRadius="{ThemeResource ControlCornerRadius}">
```

Comment:
@StephenLPeters @ranjeshj @tashatitova we can move the **CornerRadiusSetter** as well as the margin (if needed) into **focused visual groups**:
![image](https://user-images.githubusercontent.com/21356912/125015849-2be87b00-e025-11eb-8299-f6ed1530634e.png)
By doing this, we don't have to change any sizes and potentially we don't need a margin in focused state either if we can sacrifice thumb being cut off in left or right most position while the control is focused.
If we can't sacrifice this scenario, then we can also apply correct margin (2 0 2 0 or 0 2 0 2 for horizontal/vertical) to mitigate the cut off the thumb. However, it will bring unnecessary thumb movement when focused/unfocused.